docs/advanced/Concurrency.md

```diff
@@ -4,11 +4,14 @@ In the basics section, we saw how to use the helper effects `takeEvery` and `tak
 
 In this section we'll see how those helpers could be implemented using the low-level Effects.
 
+These helpers return a forked task, which creates an attached task that will block execution of the saga that invokes the helper. If you need the helpers to create a detached task, which is what the official redux-saga's implementation of `takeLatest` and `takeEvery` does, then use `spawn` instead. 
+
```

Comment:
I think ive made a mistake there and the correct version should be this
```
function takeLatest(pattern, saga, ...args) {
  return fork(function* () {
    let lastTask
    while (true) {
      const action = yield take(pattern)
      if (lastTask)
        yield cancel(lastTask) // cancel is no-op if the task has already terminated

      lastTask = yield fork(saga, ...args.concat(action))
    }
  })
}
```

Would u like to fix this in the docs? Probably there are like 3 places to correct this when describing `takeEvery`, `takeLatest` and `throttle`